src/main/java/org/springframework/samples/petclinic/vet/VetController.java

```diff
@@ -1,18 +1,7 @@
-/*
- * Copyright 2012-2019 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+// Line 1
+//Line 2
+//Line 3
+
```

Comment:
🧑‍🏫 says:

54: Changing `PageRequest.of(page - 1, pageSize)` to `PageRequest.of(page, pageSize)` incorrectly shifts the page index, which could result in fetching the wrong set of records from the database, potentially leading to empty or unexpected results.